Use FormUrlEncodedContent to build query

scythx opened the pull request.

sudheesh001 commented on LoklakDotNet/Loklak.cs:
```diff
@@ -104,20 +110,23 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
             if(fields!= null)
             {
-                qs += "&fields=" + string.Join(",", fields);
+                qs.Add("fields",string.Join(",",fields));
```
please leave a space after the `,` making it `, fields` instead of `,fields`

sudheesh001 commented on LoklakDotNet/Loklak.cs:
```diff
@@ -104,20 +110,23 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
             if(limit!=-1)
             {
-                qs += "&limit=" + limit;
+                qs.Add("limit",limit.ToString());
```
Same here, please leave a space after the `,` making it as `, limit`

sudheesh001 commented on LoklakDotNet/Loklak.cs:
```diff
@@ -104,20 +110,23 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
             if(timeZoneOffset!=-1)
             {
-                qs += "&timeZoneOffset=" + timeZoneOffset;
+                qs.Add("timeZoneOffset",timeZoneOffset.ToString());
```
Similar here too, `, timeZoneOffset...`

sudheesh001 commented on LoklakDotNet/Loklak.cs:
```diff
@@ -132,16 +141,19 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
             if (!string.IsNullOrWhiteSpace(color_text))
             {
-                qs += "&color_text=" + color_text;
+                qs.Add("color_text",color_text);
```
Same here `, color_text`

sudheesh001 commented on LoklakDotNet/Loklak.cs:
```diff
@@ -132,16 +141,19 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
             if (!string.IsNullOrWhiteSpace(color_background))
             {
-                qs += "&color_background=" + color_background;
+                qs.Add("color_background",color_background);
```
`, color_background`

sudheesh001 commented on LoklakDotNet/Loklak.cs:
```diff
@@ -87,8 +90,11 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
+            Dictionary<string,string> qs = new Dictionary<string,string>();
+
+            qs.Add("screen_name",screen_name);
+            qs.Add("followers",follower_count.ToString());
```
Change this to be `, follower_count. ...` need the spacing after the `,` so that it looks more pleasing to read this.

raefaldhia commented:
Updated

Commit:
Use FormUrlEncodedContent to build query

Before, we used string concentration method and this method is
inefficient and also unsafe to build query.The best method for this by
using FormUrlEncodedContent.

jayvdb commented:
@sudheesh001 ping.
